CONTRIBUTING.rst

```diff
@@ -12,6 +12,33 @@ Git workflow
 The WarpX project uses `git <https://git-scm.com>`_ for version control.
 If you are new to git, you can follow `this tutorial <https://swcarpentry.github.io/git-novice/>`__.
 
+
+What to do when
+---------------
```

Comment:
Looking at the overall content of the file, should we change the "level" of this header? Otherwise the subsequent git-related content falls under "What to do when", which is currently at the same level as "Git workflow").
```suggestion
What to do when
^^^^^^^^^^^^^^^
```